src/java.base/unix/native/criuengine/criuengine.c

```diff
@@ -102,9 +107,13 @@ static int checkpoint(pid_t jvm,
             "-t", jvmpidchar,
             "-D", imagedir,
             "--shell-job",
-            "-v4", "-o", "dump4.log", // -D without -W makes criu cd to image dir for logs
```

Comment:
The dump4.log is a great debugging aid for checkpoint and restore failures, please keep it.

I think a part of the reason for this change is to provide some logging to the console. I think the reporting should be implemented separately https://github.com/openjdk/crac/pull/64#discussion_r1221827050

Reply:
By default the file is still used, see lines 113 - 116. The args just support overriding those.